java/server/src/org/openqa/grid/web/Hub.java

```diff
@@ -19,6 +19,8 @@
 
 import com.google.common.collect.Maps;
 
+import com.sun.org.glassfish.gmbal.ManagedObject;
```

Comment:
These comments were just pointed out to me. This is a pretty easy fix. I'll make the appropriate updates to the other imports as well.